vuepress/docs/next/docs/curate/troubleshooting-guide.md

```diff
@@ -3,20 +3,23 @@
 ## How do I access the logs? 
 **A bundle installation or removal has failed. How do I access the logs?**
 
-Currently the Entando Component Manager (CM) logs are only available in Kubernetes through Openshift dashboards or CLI tools like K9s and kubectl.
+Currently, the Entando Component Manager (CM) logs are only available in Kubernetes through Openshift dashboards or CLI tools like K9s and kubectl.
```

Comment:
1) Simplify and flip the precedence "The Entando Component Manager (CM) logs can be viewed using CLI tools like kubectl or oc, or visualization dashboards like OpenShift or K9s."
2) Minor, OpenShift (S is capitalized)